app/controllers/businesses_controller.rb

```diff
@@ -1,6 +1,10 @@
 class BusinessesController < ApplicationController
   def show
-    @business = Business.first
+    @business = Business.find(params[:id])
     @items = ["250x300", "300x300", "200x200", "200x300"]
   end
+
+  def new  #corresponding create action is in Uesr::BusinessController
```

Comment:
I think this is a good candidate for a comment as it could be very mysterious.  I also commented the corresponding method accordingly.